### DISCUSSION TITLE
Prep for 11/9/2022: Where do we find the password?

### DISCUSSION THREAD

**Discussion by smt171** [GerryDazoo/Slinger#223] [General]
I searched and did not see this question. 

Where do we find the password (required by Nov 9, 2022 before the server shutdown).

For example, is it in a file on our computer? Or do we connect to the Sling Box to watch AND in another browser tab run a script sniffs the pw and shows us?

Can you provide step by step instructions? 

Also can you please confirm we do not need to flash the Sling Box or upgrade to a custom firmware to make this work? 

Thanks.

**Comment by lepmail** [comment]
Log in to your account, then grab this:
https://newwatchsecure.slingbox.com/watch/slingAccounts/account_boxes_js

**Reply by DVoorhees1957** [reply]
Just be aware: the above link only works for Slingboxes that are already tied to your account (via what Slingplayer calls the Directory). If your Slingbox isn't in the Directory, you're out of luck - the JS code above won't show your password, and the "official" way to add a new Slingbox to your account doesn't work any more...

**Reply by GerryDazoo** [reply]
the "official" way to add a new Slingbox to your account doesn't work any more...

I was unaware that this was the case. Are you sure?

**Reply by DVoorhees1957** [reply]
Yup... when you enter the following URL, https://www.slingbox.com/getstarted, you get an error message like the following: "The requested URL was rejected. Please consult with your administrator. Your support ID is: xxxxxxxxxxx". Click on the link yourself, and you'll see.

I've emailed what passes for Slingbox support, but they didn't respond to the fact that the page is broken - just pointed me to other KB articles that, if followed, take you back to the "get started" URL (at least for Slingbox 350s and 500s). I also asked if there was a way to get the FinderID based on having the device serial number, but they said that 350s and 500s "don't have FinderIDs" - which I know is not true, it's just that they've purposely hidden the FinderIDs on 350s and 500s. That's the primary reason I created the new discussion https://github.com/GerryDazoo/Slinger/discussions/226

**Reply by GerryDazoo** [reply]
I've just been using the SlingPLayerDesktop to do the reconfig. Which worked a couple of days ago. I do get the same error as you if I try to access it directly.
All slingboxes have finderids and their not hidden or at least not hard to find. If you don't configure a slingbox IP in my latest code it will scan the LAN and print out all the slingbox(es) IP port and FInderID.

**Reply by DVoorhees1957** [reply]
Tried to add my Slingbox 350 via Slingplayer Desktop, and it took me right back to the web-based setup URL (which is broken)...

Cool about scanning the LAN! Is there a specific command to go find all of the Slingboxes on the LAN (and their FinderIDs)? I didn't see that documented anywhere...

EDIT: Never mind, I saw that I should just use the bare-bones config.ini, which has no Slingboxes defined... found my Slingbox 350s FinderID that way!

**Reply by smt171** [reply]
So when I try that link I get something like:

http://newwatch.slingbox.com/?ticket=c3a1b1d1-123d-1234-a123-abcd1fg2 (made up but you get the point). 

It looks like a Mac address?! Is that what I want?

I looked in Settings and found the home ip and port, model, firmware and hardware version, activation date, etc. and copied to document for each. 

I don't see anywhere that says password so I'm guessing that ticket number is it?! 

I have 5 slingboxes on the same acct in different locations. (3 here, 1 at moms, 1 at inlaws). 

Will the link be different for each? Or one link period.

**Reply by smt171** [reply]
Nevermind. Got it. 

From a COMPUTER, not mobile device (phone, etc):

Step 1: login to your sling player acct: Go to
http://newwatch.slingbox.com/ and login

Step 2: run the js script in a separate browser tab...
https://newwatchsecure.slingbox.com/watch/slingAccounts/account_boxes_js

...and it prints out all of your boxes and the pws for each and finder ID.

Notes: You only have to do it once and it grabs for all of your boxes regardless of location.

**Comment by bradut** [comment]
You don't need to do anything to the SlingBox in order to use this software.
It just [magically] works !

**Comment by tom76017** [comment]
Just read the README.txt in the download,
